diff --git a/godot-core/src/builtin/collections/packed_array.rs b/godot-core/src/builtin/collections/packed_array.rs
index b876c331..2bdb7aed 100644
--- a/godot-core/src/builtin/collections/packed_array.rs
+++ b/godot-core/src/builtin/collections/packed_array.rs
@@ -13,7 +13,7 @@ use std::iter::FromIterator;
 use std::{fmt, ops, ptr};

 use godot_ffi as sys;
-use sys::{ExtVariantType, GodotFfi, SysPtr, ffi_methods};
+use sys::{ffi_methods, ExtVariantType, GodotFfi, SysPtr};

 use crate::builtin::collections::extend_buffer::ExtendBufferTrait;
 use crate::builtin::*;
@@ -45,7 +45,8 @@ pub type PackedFloat64Array = PackedArray<f64>;
 pub type PackedStringArray = PackedArray<GString>;
 pub type PackedVector2Array = PackedArray<Vector2>;
 pub type PackedVector3Array = PackedArray<Vector3>;
-#[cfg(since_api = "4.3")] #[cfg_attr(published_docs, doc(cfg(since_api = "4.3")))]
+#[cfg(since_api = "4.3")]
+#[cfg_attr(published_docs, doc(cfg(since_api = "4.3")))]
 pub type PackedVector4Array = PackedArray<Vector4>;
 pub type PackedColorArray = PackedArray<Color>;

@@ -889,7 +896,8 @@ impl_to_byte_array!(PackedFloat64Array);
 impl_to_byte_array!(PackedStringArray);
 impl_to_byte_array!(PackedVector2Array);
 impl_to_byte_array!(PackedVector3Array);
-#[cfg(since_api = "4.3")] #[cfg_attr(published_docs, doc(cfg(since_api = "4.3")))]
+#[cfg(since_api = "4.3")]
+#[cfg_attr(published_docs, doc(cfg(since_api = "4.3")))]
 impl_to_byte_array!(PackedVector4Array);
 impl_to_byte_array!(PackedColorArray);

@@ -1080,5 +1088,9 @@ impl PackedByteArray {
 }

 fn populated_or_err(array: PackedByteArray) -> Result<PackedByteArray, ()> {
-    if array.is_empty() { Err(()) } else { Ok(array) }
+    if array.is_empty() {
+        Err(())
+    } else {
+        Ok(array)
+    }
 }
